### PR TITLE
Cleanup backports

### PR DESCRIPTION
**Pull request by mhaskel**
@daenney can you double-check my squashing? I'm willing to merge if you don't want to since you did most of the code changes? either way :)

Squashed/cleanup of #492

### CODE REVIEW

File: manifests/params.pp [hunk 1]
```diff
@@ -119,7 +119,27 @@
...
+                        'repos'     => 'main contrib non-free',
+          }
+        }
+      }
+    }
     'ubuntu': {
+      $backports = {'location' => 'http://archive.ubuntu.com/ubuntu',
+                    'key'       => '630239CC130E1A7FD81A27B140976EAF437D05B5',
+                    'repos'     => 'main universe multiverse restricted',
+      }
```
Comment (daenney) [inline manifests/params.pp]: Crap, arrows aren't aligned in the `$backports` hashes. That's going to give lint errors.

File: manifests/params.pp [hunk 2]
```diff
@@ -119,7 +119,27 @@
     }
   }
   case $distid {
+    'debian': {
+      case $distcodename {
+        'squeeze': {
+          $backports = {'location' => 'http://backports.debian.org/debian-backports',
+                        'key'       => 'A1BD8E9D78F7FE5C3E65D8AF8B48AD6246925553',
+                        'repos'     => 'main contrib non-free',
+          }
```
Comment (daenney) [inline manifests/params.pp]: Crap, arrows aren't aligned in the $backports hashes. That's going to give lint errors.

File: manifests/params.pp [hunk 3]
```diff
@@ -119,7 +119,27 @@
...
+        }
+        default: {
+          $backports = {'location' => 'http://ftp.debian.org/debian/',
+                        'key'       => 'A1BD8E9D78F7FE5C3E65D8AF8B48AD6246925553',
+                        'repos'     => 'main contrib non-free',
+          }
```
Comment (daenney) [inline manifests/params.pp]: Crap, arrows aren't aligned in the $backports hashes. That's going to give lint errors.

**Comment by mhaskel** [comment]
@daenney fixed.

File: manifests/source.pp [hunk 4]
```diff
@@ -44,17 +44,22 @@
...
+      $_pin = {
+        'ensure'   => $ensure,
+        'priority' => $pin,
+        'before'   => $_before,
+        'origin'   => $host,
+      }
+    } else {
+      fail('Received invalid value for pin parameter')
     }
+    create_resources('apt::pin', { $name => $_pin })
```
Comment (daenney) [inline manifests/source.pp]: Crap. You can't use `$name` here without interpolating it for some reason. I've seen this before somewhere with `create_resources`. Changing this to `"${name}"` should fix it.

Comment (mhaskel) [inline manifests/source.pp]: that's what I get for trying to clean it up. :|

**Comment by mhaskel** [comment]
@daenney that should do it, but make sure travis works first.

**Pull request merged**